www/docs/nextjs/ssr.md

```diff
@@ -43,10 +43,9 @@ export default withTRPC<AppRouter>(
 
     return {
       url,
-      getHeaders() {
-        return {
-          'x-ssr': '1',
-        };
+      headers: {
```

Comment:
why not support a method anymore?

Reply:
it works both as a method or a plain object